apps/irys-bera-nodejs/tsconfig.json

```diff
@@ -0,0 +1,109 @@
+{
+  "compilerOptions": {
```

Comment:
Nit, but this verbose version of the `tsconfig.json` doesn't conform with the others in the monorepo. Consider reducing to just the minimum

Reply:
Yeah this is something I need to get around to because it's not applied everywhere.